_quarto.yml

```diff
@@ -239,6 +248,15 @@ website:
           contents:
             - docs/install-create-run.qmd
             - docs/debug.qmd
+        - section: "Tutorials"
+          contents:
+            - tutorials/index.qmd
```

Comment:
Let's just link to the one tutorial here and save the listing page for follow up